integration-tests/validator/src/main/java/org/apache/camel/quarkus/component/validator/it/ValidatorResource.java

```diff
@@ -32,28 +33,12 @@ public class ValidatorResource {
     @Inject
     ProducerTemplate producerTemplate;
 
-    @Path("/classpath")
+    @Path("/validate/{scheme}")
     @POST
     @Consumes(MediaType.APPLICATION_XML)
     @Produces(MediaType.TEXT_PLAIN)
-    public String processOrderInXmlFromClassPath(String statement) {
-        return producerTemplate.requestBody("direct:classpath", statement, String.class);
-    }
-
-    @Path("/filesystem")
-    @POST
-    @Consumes(MediaType.APPLICATION_XML)
-    @Produces(MediaType.TEXT_PLAIN)
-    public String processOrderInXmlFromFileSystem(String statement) {
-        return producerTemplate.requestBody("direct:filesystem", statement, String.class);
-    }
-
-    @Path("/http")
-    @POST
-    @Consumes(MediaType.APPLICATION_XML)
-    @Produces(MediaType.TEXT_PLAIN)
-    public String processOrderInXmlFromHTTP(String statement) {
-        return producerTemplate.requestBody("direct:http", statement, String.class);
+    public String processOrderInXmlFromFileSystem(String statement, @PathParam("scheme") String scheme) {
```

Comment:
`processOrderInXmlFromFileSystem` => Don't we do more than file system ?

Reply:
Thanks just renamed to a generic one.